packages/jbrowse-plugin-apollo/src/components/ImportFeatures.tsx

```diff
@@ -46,6 +54,64 @@ export function ImportFeatures({
     )
   }
 
+  function handleDeleteFeatures(e: React.ChangeEvent<HTMLInputElement>) {
+    setDeleteFeatures(e.target.checked)
+  }
+
+  // fetch and set features count for selected assembly
+  useEffect(() => {
+    if (assemblyId) {
+      updateFeaturesCount()
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [assemblyId])
+
+  function internetAccount() {
+    const apolloInternetAccounts = internetAccounts.filter(
+      (ia: { type: string }) => ia.type === 'ApolloInternetAccount',
+    ) as ApolloInternetAccountModel[]
+
+    if (apolloInternetAccounts && apolloInternetAccounts.length) {
+      return apolloInternetAccounts[0]
+    }
+  }
```

Comment:
We need to make sure the internet account is the same one that the selected assembly comes from, since there may be more than one ApolloInternetAccount. Something like this: https://github.com/GMOD/Apollo3/blob/1977709f1a40032cdbe40cf9ed7292a6d939d573/packages/jbrowse-plugin-apollo/src/BackendDrivers/CollaborationServerDriver.ts#L21-L50